### ISSUE TITLE
How to enable Sigmoid Activation for fine-tuning the BERT / DistilBERT pre-trained models?

### ISSUE THREAD

**Issue by katreparitosh** [amaiya/ktrain#302]
Hello,

I have a dataset with two columns - The first column contains the **text** and the **label** column contains the labels for the sentiment (either positive or negative).

When I utilize the **predict_proba()** function for inference, it outputs the probability score for both the classes. 

See below.
![image](https://user-images.githubusercontent.com/42617598/102962480-fa86e280-450c-11eb-88a8-9b3d0cb9a6b3.png)

I believe Hugging Face / BERT by _default_ uses **Softmax Activation**. See [here](https://github.com/amaiya/ktrain/blob/d0c24635c42faddd5b7e55be261712e25410ca67/ktrain/text/models.py#L276) and [here](https://github.com/amaiya/ktrain/blob/d0c24635c42faddd5b7e55be261712e25410ca67/ktrain/text/models.py#L249).

My requirement is to use binary **Sigmoid Activation** output that would give a probability between 0 - 1.

`"probability" = "0.82.."`

@amaiya It would be great if you could help on how it can be achieved using the amazing Ktrain! 

Thanks.

**Comment by amaiya** [comment]
Hello,

First, the probabilities returned when `return_proba=True` should add up to 1, as demonstrated in the [tutorial/example notebooks](https://nbviewer.jupyter.org/github/amaiya/ktrain/blob/master/tutorials/tutorial-04-text-classification.ipynb).  

Second, you don't need to do anything special to get the probabilities you want.  This is also demonstrated in the tutorial notebooks.  For binary classification problems,  you can just take the probability of the class of interest:
```python
probs = predictor.predict(doc, return_proba=True)
print(probs)
# should print something like array([0,18, 0.82], dtype=float32)
probability = probs[1] # probability of document being the "positive" class (class_id=1)
```

**ktrain** configures the model automatically  based on how the data is preprocessed.  Since, the data is 1-hot-encoded by default for binary/multiclass problems, the models use softmax and categorical_crossentropy (for binary/multiclass problems).  For multilabel problems, the models are configured differently.

**Comment by katreparitosh** [comment]
Thanks a lot!
